frontend/components/AddHostsModal/PlatformWrapper/MacosPanel/_styles.scss

```
@@ -0,0 +1,6 @@
+.macos-panel {
+  &__enroll-link input {
+    font-family: "SourceCodePro", $monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "SourceCodePro" (font-family-name-quotes)

(font-family-name-quotes)